my-app/src/components/Languages/translation.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component, Suspense } from 'react';
+import { withTranslation } from 'react-i18next';
+
+class LegacyComponentClass extends Component {
```

Comment:
look like leftover from some example. I believe this file can be removed.

Reply:
yes, this was an example for use "t". Deleted :)